Sources/BraveWallet/Crypto/Transactions/TransactionDetailsView.swift

```diff
@@ -14,122 +14,569 @@ struct TransactionDetailsView: View {
+              backgroundColor: Color(braveSystemName: .gray40),
+              iconColor: Color.white,
+              length: assetIconLength,
+              maxLength: maxAssetIconLength
+            )
+            Image(braveSystemName: "leo.arrow.right")
+            GenericAssetIconView(
+              backgroundColor: Color(braveSystemName: .gray20),
+              iconColor: Color.black,
+              length: assetIconLength,
+              maxLength: maxAssetIconLength
+            )
           }
-          detailRow(title: Strings.Wallet.transactionDetailsDateTitle, value: dateFormatter.string(from: transactionDetailsStore.transaction.createdTime))
-          if !transactionDetailsStore.transaction.txHash.isEmpty {
-            Button(action: {
-              if let txNetwork = self.networkStore.allChains.first(where: { $0.chainId == transactionDetailsStore.transaction.chainId }),
-                 let url = txNetwork.txBlockExplorerLink(txHash: transactionDetailsStore.transaction.txHash, for: txNetwork.coin) {
-                openWalletURL(url)
+        } else {
+          VStack {
+            HStack {
+              if let fromToken {
+                AssetIconView(
+                  token: fromToken,
+                  network: parsedTransaction.network,
+                  length: assetIconLength,
+                  maxLength: maxAssetIconLength
+                )
+              } else {
+                GenericAssetIconView(length: assetIconLength, maxLength: maxAssetIconLength)
               }
-            }) {
-              detailRow(title: Strings.Wallet.transactionDetailsTxHashTitle) {
-                HStack {
-                  Text(transactionDetailsStore.transaction.txHash.truncatedHash)
-                  Image(systemName: "arrow.up.forward.square")
+              if let fromAmount {
+                if let fromToken {
+                  Text(verbatim: "\(fromAmount) \(fromToken.symbol)")
+                } else {
+                  Text(verbatim: fromAmount)
                 }
-                  .foregroundColor(Color(.braveBlurpleTint))
               }
             }
-            .listRowBackground(Color(.secondaryBraveGroupedBackground))
-          }
-          if let network = transactionDetailsStore.network {
-            detailRow(title: Strings.Wallet.transactionDetailsNetworkTitle, value: network.chainName)
-          }
-          detailRow(title: Strings.Wallet.transactionDetailsStatusTitle) {
-            HStack(spacing: 4) {
-              Image(systemName: "circle.fill")
-                .foregroundColor(transactionDetailsStore.transaction.txStatus.color)
-                .imageScale(.small)
-                .accessibilityHidden(true)
-              Text(transactionDetailsStore.transaction.txStatus.localizedDescription)
-                .foregroundColor(Color(.braveLabel))
-                .multilineTextAlignment(.trailing)
+            Image(braveSystemName: "leo.arrow.right")
```

Comment:
since it's always gonna be inside a 1`VStack` should the image be `leo.arrow.down`?

Reply:
We're still using the right arrow on desktop (& in designs); I think best to keep right arrow for consistency? (Arrow is on separate line on mobile due to screen space)

![desktop](https://github.com/brave/brave-ios/assets/5314553/bad2cde4-a23d-4354-be91-b622084a5119)